source/github/github.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	defaultURL = "https://github.com"     // Default GitHub URL
-	defaultAPI = "https://api.github.com" // Default GitHub API URL
+	defaultURL = "https://github.com/"     // Default GitHub URL
```

Comment:
When defaultURL and source url (from docker-compose.yml) are not same, we construct auth api url using source url which results in 404 error.
https://github.com/go-vela/server/blob/e20520b752adef0f96bfeffb4bab148468ef3137/source/github/github.go#L59

**defaultAPI should be used to authorize the Github user.**